rust_src/src/symbols.rs

```diff
@@ -170,3 +175,19 @@ fn indirect_variable_lisp(object: LispObject) -> LispObject {
         object
     }
 }
+
+
+/// Make SYMBOL's value be void.
+/// Return SYMBOL.
+#[lisp_fn]
+fn makunbound(symbol: LispObject) -> LispObject {
+    if let Some(sym) = symbol.as_symbol() {
```

Comment:
Would this not be better as:

    let sym = symbol.as_symbol_or_error();
    if sym.is_contant() {
        xsignal!(Qsetting_constant, symbol);
    }
    unsafe {
        Fset(symbol.to_raw(), Qunbound);
    }
    symbol

Reply:
thanks, updated